PowerShellGet/PSModule.psm1

```diff
@@ -10609,17 +11048,46 @@ function Install-PackageUtility
                     }
                     else
                     {
-                        if($InstalledModuleInfo.Version -lt $version)
+                        $installedModuleVersion = $InstalledModuleInfo.Version.ToString()
+                        if ((Get-Member -InputObject $InstalledModuleInfo -Name PrivateData -ErrorAction SilentlyContinue) -and `
+                            ($InstalledModuleInfo.PrivateData.ContainsKey('PSData')) -and `
+                            ($InstalledModuleInfo.PrivateData.PSData.ContainsKey('Prerelease')))
+                        { 

```

Comment:
Consider adding checks for Hashtable type similar to other places.

Reply:
This is the root cause for "InvokeMethodOnNull,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage" you are getting.

---
In reply to: [141772563](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r141772563) [](ancestors = 141772563)

Reply:
Thank you Manikyam, that fixed it!

---
In reply to: [141799134](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r141799134) [](ancestors = 141799134,141772563)